nodejs/init.sls

```diff
@@ -1,10 +1,37 @@
-node-pkgs:
-  pkg:
-    - installed
-    - names:
-      - npm
-      - nodejs-legacy
```

Comment:
We might want to add a state that uninstalls `npm` and `nodejs-legacy`, unless the `nodejs` package from the NodeSource PPA will do that automatically?